build.gradle

```diff
@@ -18,18 +18,28 @@ dependencies {
 
     //implementation 'com.github.signum-network:signumj:v1.0.0'
     implementation 'com.github.signum-network:signumj:f2522b5480'
-    
+
     implementation 'io.reactivex.rxjava2:rxjava:2.2.15'
 
     implementation 'com.github.jiconfont:jiconfont-swing:1.0.0'
     implementation 'com.github.jiconfont:jiconfont-font_awesome:4.7.0.1'
-    
+
     //logging
 	implementation 'org.apache.logging.log4j:log4j-api:2.15.0'
 	implementation 'org.apache.logging.log4j:log4j-core:2.15.0'
 
     // Use JUnit test framework
     testImplementation 'junit:junit:4.12'
+
+	// lombok
```

Comment:
I'm not familiar with this framework, since I'll probably have to keep it later alone, I'm not willing to add it.

Reply:
lombok to have more simple / cleaner / more readable code. Have a look at it, it's very convenient. 100% safe (just shortcut to straight java code)

Reply:
meanwhile I'll remove it